src/applications/mhv-medical-records/components/RecordList/AllergyListItem.jsx

```diff
@@ -13,13 +13,13 @@ const AllergyListItem = props => {
       data-testid="record-list-item"
     >
       {/* web view header */}
-      <h3 className="vads-u-font-size--h4 vads-u-margin-bottom--0p5 vads-u-line-height--4 no-print">
+      <h5 className="vads-u-font-size--h5 vads-u-margin-y--1 vads-u-line-height--4 no-print">
```

Comment:
The previous header tag is in RecordListItem and is an `<h2>`. Structurally an HTML page cannot have an `<h5>` directly after an `<h2>`

The instruction is "All links in cards should be styled as a regular link, bold (not as a header)". I have two questions:
1. UCD said to style as a regular link, not as an `<h5>`. Is the h5 styling close enough or do we need to style this as a regular paragraph text?
2. Does UCD want this to remain a header tag at all, for screen readers? Or should that go away?